drivers/scheduler/dcos/dcos.go

```diff
@@ -394,6 +394,16 @@ func (d *dcos) Describe(ctx *scheduler.Context) (string, error) {
 	return "", nil
 }
 
+func (d *dcos) ScaleApplication(ctx *scheduler.Context, deploymentScaleFactorMap map[string]int32) error {
```

Comment:
nit: call the variable just scaleFactorMap.. deployment is a kubernetes specific term

Reply:
Fixed.

Reply:
missed this one